### DISCUSSION TITLE
Nag Screen for Supporter Certificate in SB Classic

### DISCUSSION THREAD

**Discussion by uDEV2019** [sandboxie-plus/Sandboxie#2345] [General]
Could someone explain why there is a nag screen build into Sandboxie Classic Version?
![sb_classic](https://user-images.githubusercontent.com/51911097/195957072-7a355cee-dc54-4465-9228-eb21778a4d82.png)

This is Classic-Version v5.59.2, that got released on October 10 and got installed one day later.

The nag screen appeared around 10 or 15 minutes after i run an app in sandboxed mode.
The runned app was closed around 5 minutes before the nag sceen appeared.
On that nag screen, the three buttons appear after around 5 seconds.

As staited in readme.md the Classic Version is licensed under GNU GPL 3.0.
Nagware is prohibited under this license.

@DavidXanatos Please give some explaination to this.

**Comment by DavidXanatos** [comment]
@uDEV2019 
GNU GPL 3.0 only stipulates that the code must be open source, what the code does is not scope of the license.
You are free (like in freedom not like in free beer) to build your won fork without that info screen.

The purpose of that screen is to remind you that software development is a lot of work and that decent people should reward those whose work they are benefiting from, or even better contribute to that work directly.

You can earn a Contribute certificate by helping with documentation, translations or development of the project.

**Comment by uDEV2019** [comment]
Thanks for answering.
A "Info screen" would imply, the screen is can be closed without waiting a counter to get counted down to zero.

I do know, that development costs time. Therefor it also costs money and if the software is under professional (business) development i would understand such nag screen. On the other hand on some open source projects that are released by businesses, you mainly pay for usage of support.

On other projects i contributed translations (e.g your priv10) and i'll do that for free as i'll do it in my free time.

I hope you'll defuse the nag screen for the classic strain sometime.
